lib/jinterface/java_src/com/ericsson/otp/erlang/OtpEpmd.java

```diff
@@ -242,7 +241,7 @@ private static int r4_lookupPort(final AbstractNode node)
                 System.out.println("<- (no response)");
             }
             throw new IOException("Nameserver not responding on " + node.host()
-                    + " when looking up " + node.alive());
+                    + " when looking up " + node.alive(), e);
```

Comment:
Ok, so it's been discussed and Java 1.6 will be required from v18 and the tests will run nicely after the test servers get updated.